pkg/integrations/windows_exporter/config.go

```diff
@@ -0,0 +1,7 @@
+// +build !windows
+
+package windows_exporter //nolint:golint
+
+// Config placeholder config for dummy windows_exporter
+type Config struct {
+}
```

Comment:
This will cause problems if it's configured on non-Windows. Can you make the config struct cross-platform? You can make the ApplyConfig method Windows-only by putting it in its own file.